src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -32,21 +33,26 @@ class GcsRpcClient;
 /// Executor saves operation and support retries.
 class Executor {
  public:
-  explicit Executor(GcsRpcClient *gcs_rpc_client) : gcs_rpc_client_(gcs_rpc_client) {}
+  explicit Executor(GcsRpcClient *gcs_rpc_client,
```

Comment:
`explicit` should be unnecessary. Also, take `const ray::Status &` in the `abort_callback`?